lighthouse-core/gather/gatherers/animated-elements.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Gatherer = require('./gatherer.js');
+const TraceProcessor = require('../../lib/tracehouse/trace-processor.js');
+const pageFunctions = require('../../lib/page-functions.js');
+
+/* eslint-env browser, node */
+
+/**
+ * @this {HTMLElement}
+ * @param {number} nodeId
+ * @return {LH.Artifacts.AnimatedElement | undefined}
+ */
+/* istanbul ignore next */
+function getNodeData(nodeId) {
+  const elem = this.nodeType === document.ELEMENT_NODE ? this : this.parentElement; // eslint-disable-line no-undef
```

Comment:
what does eslint think is undefined here?

Reply:
I thought it might find `document` undefined similar to [this](https://github.com/GoogleChrome/lighthouse/blob/3efbf94c96651a936ce64d81314c100c58ad86a1/lighthouse-core/gather/gatherers/trace-elements.js#L28).

You are correct though since lint does not report this line when the disable comment is removed.  For future reference, why is `document` not undefined here?

Reply:
n/a moved to other pr